packages/server/database/migrations/20201130124200-removeSlackUserAccessToken.ts

```diff
@@ -0,0 +1,23 @@
+import {R} from 'rethinkdb-ts'
```

Comment:
can you add another migration for removing all the slack auths?
to determine if something should be a migration vs. mutation, i like to ask what would happen to self-hosted customers. i don't want to have to ask all of them to run a mutation, they should just be able to upgrade to the latest version & have everything work for them. so, in that case, removing the old bad auths should be a migration.

same goes for deploying to dev/staging/prod. when we upgrade, everything should (ideally) work without having to remember to run a mutation

Reply:
Nice, that's a helpful way of thinking about it. Migration added!